confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/input/ConfluenceXMLPackage.java

```diff
@@ -1060,7 +1117,8 @@ private Object readProperty(XMLStreamReader xmlReader) throws XMLStreamException
             || propertyClass.equals("SpaceDescription") || propertyClass.equals("Labelling")
```

Comment:
Starting to be quite a `if`, I guess we will need to start consider introduce a static Set with all those classes :)